### ISSUE TITLE
Blue/white color combination color contrast failure

### ISSUE THREAD

**Issue by EmmaDawsonDev** [danielcranney/profileme.dev#21]
The blue/white color combination does not reach an adequate color contrast of 4.5:1 to be accessible.
